act-rules-format.bs

```diff
@@ -512,13 +512,33 @@ Definitions {#definitions}
     </ul>
   </dd>
 
+  <dt><dfn>meet expectations</dfn></dt>
+  <dd>
+    <p>A [=test target=] meets an expectation, when that expectation is true for the test target individually, or when the test target increases a count that is required for the expectation to be true.</p>
+    <div class=example>
+      <p>**Example:** An HTML `img` element an `alt` attribute meet the following expectation:</p>
+      <blockquote>Each test target has an alt attribute</blockquote>
+    </div>
+    <div class=example>
+      <p>**Example:** An HTML element with a duplicate ID does not meet the following expectation:</p>
+      <blockquote>No test target has the same ID</blockquote>
+    </div>
+    <div class=example>
+      <p>**Example:** An HTML `video` element without captions does not meet the following expectation:</p>
```

Comment:
How do you know if this video is in the 80% or 20%?

Reply:
Following the ACT TF discussion on 31 January 2019, I suggest we change this example to:

> A set of 3 HTML 'video' elements of which 2 do not have captions does not meet the following expectation: